### DISCUSSION TITLE
GIF

### DISCUSSION THREAD

**Discussion by kdill00** [blakeblackshear/frigate#15269] [Ideas]
Hey guys, I just wanted to see if there is anyway in the future to add a way to increase the quality of the preview gif or to make a higher quality event gif at some point.  Reason being, my father was born mostly deaf and has hearing aids.  With the iphone, any mp4 clip notification when opened, takes over any sound or audio being channeled through the phone. As his hearing aids are connected with them, it cuts out anything he is listening to when he expands and views them. I really like using the preview gif as this solves the problem for him completely.  I am just curious if it’s possible to increase the quality or if maybe there is something I had not considered in phone settings to prevent mp4 notifications from acting like they are playing audio.  I can tell you guys are getting super busy here.  I appreciate the mostly thankless job you guys do as always!

**Comment by NickM-27** [comment]
Yes, you can increase the preview quality by setting the config 

```
record:
  previews_quality: high
```

But I'm not sure that's related to audio at all, so it's not clear if that will solve the issue.

**Reply by kdill00** [reply]
I have this setting on the highest but the image is still very rough. It seems when a notification is sent as an image format such as gif, it does not stop whatever is playing on the phone if opened and watched.  Example screenshots of quality from a gif vs mp4 notification respectively. 
![IMG_1844](https://github.com/user-attachments/assets/cae0adbb-fba7-4345-9dbd-49d9a410e5b2)
![IMG_1845](https://github.com/user-attachments/assets/de701c71-357a-43a4-9e33-2f396fa1a1be)

**Reply by NickM-27** [reply]
Do you mean the preview MP4 or the full resolution recording MP4? Because those are very much not comparable and it wouldn't make sense to have previews be even close to that level of quality
